style/js/shield_defs.js

```diff
@@ -206,6 +206,17 @@ export function loadShields(shieldImages) {
     colorLighten: "#613214",
   };
 
+  shields["US:AK"] = {
+    backgroundImage: shieldImages.shield40_us_ak_2,
```

Comment:
I'd suggest removing the `_2` from the SVG filename to make it consistent with other shields that have only a single shield